src/main/java/com/ericsson/ei/handlers/EventToObjectMapHandler.java

```diff
@@ -83,28 +83,29 @@ public void setJmesPathInterface(JmesPathInterface jmesPathInterface) {
 
     public void updateEventToObjectMapInMemoryDB(RulesObject rulesObject, String event, String objectId) {
         String eventId = getEventId(rulesObject, event);
-        String condition = "{\"_id\" : \"" + eventId + "\"}";
-        ArrayList<String> list =  getEventToObjectList(eventId);
-        boolean firstTime = list.isEmpty();
-        list = updateList(list, eventId, objectId);
-        ObjectMapper mapper = new ObjectMapper();
-        JsonNode entry = null;
-
+        String condition = "{\"_id\" : \"" + objectId + "\"}";
+        LOGGER.debug("Checking document exists in the collection with ID : {}", condition );
```

Comment:
For easier debugging, these two lines could be one log entry.

Reply:
have changed it to single statement.